data/styles/clear/include/priorities_4_overlays.prio.txt

```diff
@@ -1222,17 +1222,20 @@ shop-tobacco                                        # icon z18- (also has captio
 shop-trade                                          # icon z18- (also has caption(optional) z18-)
 shop-travel_agency                                  # icon z18- (also has caption(optional) z18-)
 shop-watches                                        # icon z18- (also has caption(optional) z18-)
-shop-wholesale                                      # icon z18- (also has caption(optional) z18-)
+shop-wholesale                                      # icon z17- (also has caption(optional) z17-)
 === 500
 
 amenity-bbq                                         # icon z18- (also has caption(optional) z18-)
 amenity-internet_cafe                               # icon z18- (also has caption(optional) z19-)
 amenity-public_bookcase                             # icon z18- (also has caption(optional) z18-)
 amenity-toilets                                     # icon z18- (also has caption(optional) z18-)
-craft-handicraft                                    # icon z18- (also has caption(optional) z18-)
-craft-hvac                                          # icon z18- (also has caption(optional) z18-)
-craft-key_cutter                                    # icon z18- (also has caption(optional) z18-)
-craft-locksmith                                     # icon z18- (also has caption(optional) z18-)
+craft-caterer                                       # icon z17- (also has caption(optional) z18-)
+craft-grinding_mill                                 # icon z17- (also has caption(optional) z18-)
+craft-handicraft                                    # icon z17- (also has caption(optional) z18-)
+craft-hvac                                          # icon z17- (also has caption(optional) z18-)
+craft-key_cutter                                    # icon z17- (also has caption(optional) z18-)
+craft-locksmith                                     # icon z17- (also has caption(optional) z18-)
+leisure-outdoor_seating                             # icon z18- and caption z17 (also has caption(optional) z18-)
```

Comment:
please remove the lone caption from z17

(its inherited from `area|z17-[leisure],` in Basemap_label.mapcss)

you can disable it via
```
node|z17[leisure=outdoor_seating],
{text: none;}
```

or maybe remove caption from outdoor seating completely!